src/main/java/bowling/domain/score/OpenScore.java

```diff
@@ -0,0 +1,43 @@
+package bowling.domain.score;
+
+import bowling.domain.DownedPinCount;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OpenScore implements Score {
+
+	private int remainExtraCount = 0;
+	private List<DownedPinCount> accumulatedCount;
+	private Score accumulated;
+
+	public OpenScore(DownedPinCount first, DownedPinCount second, Score accumulated) {
+		List<DownedPinCount> list = new ArrayList<>();
+		list.add(first);
+		list.add(second);
+		this.accumulatedCount = list;
+		this.accumulated = accumulated;
+	}
+
+	@Override
+	public boolean isRemainExtra() {
+		return remainExtraCount > 0;
+	}
+
+	@Override
+	public void addExtraCount(DownedPinCount count) {
+		if(isRemainExtra()) {
+			accumulatedCount.add(count);
+			remainExtraCount--;
+		}
+	}
+
+	@Override
+	public DownedPinCount getValue() {
+		if(isRemainExtra()) {
+			return DownedPinCount.NO_PIN_DOWN;
+		}
+		DownedPinCount sum = accumulatedCount.stream().reduce(DownedPinCount.NO_PIN_DOWN, DownedPinCount::sum);
```

Comment:
스트림 체인을 하다 보면 한 라인 자체가 굉장히 길어지는 모습을 볼 수 있는데요, 
이 때 적절히 개행해주시면 조금 더 가독성있는 코드가 완성될 것 같아요. 

```suggestion
        DownedPinCount sum = accumulatedCount.stream()
                                                                            .reduce()....